src/frontend/components/Select.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import './style.scss'
+const Select =(props)=> {
+   return (
+         <select name ={props.name} className="blinkas-black" onChange={props.onChange}>
```

Comment:
Hardcoding the classname entry is not right. That should come from the props. If there are any specific styles for this component, move it to the `styles.scss` file.